.github/workflows/branch-protection-audit.yml

```
@@ -0,0 +1,68 @@
+
+# Deliberately NOT a required status check. It audits branch protection
+# itself, so making it required would let the exact failure mode it exists
+# to catch — required_status_checks silently losing an entry — also disable
+# this workflow's ability to report on it. See CLAUDE.md's "Required status
+# checks and the release bot" section: `release` carries five required
+# contexts split across two layers (a ruleset and classic branch
+# protection), and a ruleset-only migration would silently drop
+# `security-suites` because only four of the five are duplicated there.
+permissions:
+  contents: read
+  administration: read
```

actionlint (1.7.12)

[error] unknown permission scope "administration". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)